### PR TITLE
github actions pinned to go 1.13

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

## Motivation and Context

Solves #282.
- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

Screen Shot 2020-11-09 at 19 29 18
Screen Shot 2020-11-09 at 19 29 33

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/only-ci.yaml [hunk 1]
```diff
@@ -8,8 +8,19 @@ on:
...
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ 1.13.x ]
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
     steps:
+      - uses: actions/checkout@master
```
Comment (Waterdrips) [inline .github/workflows/only-ci.yaml:17]: are we supposed to have 2x checkouts? (https://github.com/alexellis/k3sup/pull/283/files#diff-df63d808e4fa88b92c966c08f5d9bfacf58325695381ed547390fc52eb39360cR24)

Comment (developer-guy) [reply .github/workflows/only-ci.yaml:17]: we already used actions/checkout@master step in [faas-netes workflow](https://github.com/openfaas/faas-netes/blob/master/.github/workflows/build.yaml), should I update it ?

Comment (Waterdrips) [reply .github/workflows/only-ci.yaml:17]: its the checkout master on below your added section im asking about, line 24-26. We checkout here, (line 17) then setup go (line 20) then re-checkout master - not sure we need both?

Comment (developer-guy) [reply .github/workflows/only-ci.yaml:17]: done, thank you. It's my mistake, we don't need both, you're right.

**Review by Waterdrips — APPROVED** [review]
LGTM

File: .github/workflows/only-ci.yaml (hunk 1 above)
Comment (developer-guy) [reply .github/workflows/only-ci.yaml:17]: I updated files according to your review 👍

Comment (alexellis) [reply .github/workflows/only-ci.yaml:17]: Do we have the same issue with the arkade PR?

Comment (developer-guy) [reply .github/workflows/only-ci.yaml:17]: Yep, I opened the same issue for the k3sup .

**Comment by alexellis** [comment]
@developer-guy we have a rebase error, could you take a look? It may be that only-ci was renamed to build.yaml

**Comment by alexellis** [comment]
Please compare the contents of the PR to build.yaml and decide whether we can close this PR?

**Comment by developer-guy** [comment]
I am looking into this 👍

**Comment by developer-guy** [comment]
@alexellis done, thank you:

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**